firmware/rhizome.ino

```diff
@@ -53,22 +53,32 @@ unsigned long lastUpdate = millis();
 /**
  * Does any preliminary setup and initializations before the Rhizome starts the operation loop.
  */
+// Setting the photon to semi automatic mode, which means it 
+// does not connect to WiFi until Particle.connect() is called.
+SYSTEM_MODE(SEMI_AUTOMATIC);
 void setup() {
     //initialize the Dallas Onewire bus pin - Digital 0
     ow_setPin(D0); //This should later be accomplished by equipment setup OR constructor
 
     // Turn on the display
     screen.initScreen();
+    
+    //Turn on for debugging
+    //Serial.begin(9600);
+    
+    delay(1000);
 }
 
 /**
  * The bulk of the program. Runs repeatedly until the Rhizome is powered off.
  */
 void loop() {
+
     //call work on all installed equipment in the sprouts list
     sproutList.work();
+ 
 
-    //refresh the display
+	screen.captureButtonPress();
```

Comment:
I haven't downloaded this yet, so I'm not sure if you're using tabs or what, but make sure the indentation matches up for style. Pretty sure we're doing 4 space indents in this project judging by the other lines shown in the PR (though I could have sworn we used 2 spaces... hmm.)